packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -159,10 +159,15 @@ export class TspLanguageClient {
         logger.error(
           [
             `TypeSpec server executable was not found: '${this.exe.command}' is not found. Make sure either:`,
+            this.exe.command === "node"
```

Comment:
I thikn for this one is ok as in node code the extension is auto resolved later.